src/layers/attention.h

```diff
@@ -463,8 +463,9 @@ class Attention {
                     auto srcV = value.Row(b * ctx->inputSeqLen + seq) + h * headSize;
                     auto dstV = presentValue.getSequence(pastSeqLen + seq, b, h);
 
-                    xft::copy(dstK, srcK, headSize);
-                    xft::copy(dstV, srcV, headSize);
+                    // Suppose dstK and dstV are the same data type
```

Comment:
Since there are type conversions in storeKVCache. The old comments can be removed.